src/Sylius/Component/Promotion/Generator/PromotionCouponGeneratorInstructionInterface.php

```diff
@@ -13,32 +13,20 @@
 
 namespace Sylius\Component\Promotion\Generator;
 
-interface PromotionCouponGeneratorInstructionInterface
+interface PromotionCouponGeneratorInstructionInterface extends PromotionCouponGeneratorInstructionReadInterface
```

Comment:
Do we consider/recommend using this interface? If not, maybe we should deprecate it?

Reply:
Yes, we were discussing this with @NoResponseMate, and he mentioned that he would create a task for it.